languages.json

```diff
@@ -0,0 +1,12 @@
+{
+  "js": {
+    "extension": ".js",
```

Comment:
let's put the . in the code and make the config just js/py/etc

Reply:
fixed